emulators/qemu/Portfile

```diff
@@ -87,11 +87,7 @@ if {"${configure.build_arch}" eq "arm64"} {
 configure.args          --cpu=${host_cpu} \
                         --cc=${configure.cc} \
                         --objcc=${configure.objc} \
-                        --host-cc=${configure.cc} \
-                        --sphinx-build=${prefix}/bin/sphinx-build-${py_branch}
-
-# Use MacPorts meson, which includes platform-specific fixes
-configure.args-append   --meson=${prefix}/bin/meson
+                        --host-cc=${configure.cc}
```

Comment:
@herbygillot Oops, apparently I wrote this as a pending comment but forgot to actually submit it.

Reply:
I did a bit more digging and if I understand [the scripts](https://github.com/qemu/qemu/blob/v8.2.0/python/scripts/mkvenv.py#L955) correctly, the QEMU build system will prefer meson from the system only if it is installed as a site-package for the given python interpreter. Otherwise it falls back to the builtin vendored meson wheel.
The build system will not need the `${prefix}/bin/meson` symlink, so this means we should actually depend on `py312-meson` instead of `meson`.

Reply:
That's a very good point. I've updated the PR to remove the `meson` Python patch, and replaced `meson` with `py312-meson` as a build dependency.